dist/leaflet.distortableimage.css

```diff
@@ -5,6 +5,13 @@
   overflow: clip;
 }
 
+.l-container {
```

Comment:
@rexagod have you tried inverting the colors here to make it white and black? Just to follow the color scheme of the rest of the page? for ex: 
<img width="183" alt="Screen Shot 2019-04-12 at 9 48 29 PM" src="https://user-images.githubusercontent.com/41092741/56073073-bff70980-5d6c-11e9-9520-d00537ba95a8.png">

created by 

```
 background-color: rgb(255, 255, 255, 0.75);
  color: black;
```
Not married to how this looks - maybe we can worry about styling in a different PR. I just like the idea of consistency. But yeah i'd say that, maybe get rid of the bullets, increase the font-size it'll look better. But for now I guess let's just keep black if styling it in white in this PR would be a hassle. Up to you!

Reply:
![image](https://user-images.githubusercontent.com/33557095/56077377-06924580-5df9-11e9-9f6c-4e91b119d5cf.png)

@sashadev-sky Is this okay?